free-1.md

```diff
@@ -57,12 +99,39 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+_Most would probably be language design decision based on the semantics
+of building the language. Hopefully once the strcture of the language
```

Comment:
I think working on the systems aspects, such as coordinating dependent tasks between lists/people, will end up being rather challenging as well (from a language design and systems perspective)